src/core/account/account-files.js

```diff
@@ -75,7 +75,7 @@ function loadWalletList(
     const walletStates = {}
 
     files.forEach(file => {
-      const { SortIndex, Archived, BitcoinSeed, MK, SyncKey } = file.json
+      const { SortIndex, Archived, Hidden, BitcoinSeed, MK, SyncKey } = file.json
```

Comment:
This is the legacy Airbitz wallet list, which won't ever have `Hidden` support, since that app is deprecated. We should just default `hidden` to `false` in this routine.